v22.2/install-cockroachdb-mac.md

```diff
@@ -21,8 +21,17 @@ docs_area: deploy
 
 Use one of the options below to install CockroachDB.
 
+{% capture arch_note_homebrew %}For CockroachDB v22.x and above, Homebrew installs binaries for your system architecture, either Intel or ARM ([Apple Silicon](https://support.apple.com/en-us/HT211814)). ARM binaries are **experimental** and are not qualified for production use.
+
+For CockroachDB v21.x and below, Homebrew installs Intel binaries. Intel binaries can run on ARM systems, but with a significant reduction in performance.{% endcapture %}
+
+{% capture arch_note_binaries %}For CockroachDB v22.x and above, download the binaries for your system architecture, either Intel or ARM ([Apple Silicon](https://support.apple.com/en-us/HT211814)). ARM binaries are **experimental** and are not qualified for production use.
+
+For CockroachDB v21.x and below, download Intel binaries. Intel binaries can run on ARM systems, but with a significant reduction in performance.{% endcapture %}
```

Comment:
This should be for `v22.2.x` and above and `v22.1.x` and below. We don't intend to backport this change to 22.1.x.